squad/core/migrations/0078_user_subscriptions.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
this migration needs to be renamed. since you started working on this, new migrations were added, so I get this when trying it locally:

```
$ ./manage.py migrate
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0078_user_subscriptions, 0079_init_cache_test_run_counts in core).
To fix them run 'python manage.py makemigrations --merge'
```

you need to renumber it as `0080`, and make it depend on `0079_init_cache_test_run_counts`. just deleting it and re-running `./manage.py makemigrations` should DTRT.

Reply:
also please rebase on master to keep the PR branch clean